packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -69,15 +76,22 @@ export const SubscriptionItem = ({
     customerSubscription.cancel_at_period_end === false &&
     !((total || total === 0) && period_start)
   ) {
+    const ariaLabelledBy = "invoice-not-found-header";
+    const ariaDescribedBy = "error-product-plan-not-found-description";
```

Comment:
Q: Can you confirm this one?
I think this is the only instance I saw that has a different base `ariaLabelledBy` vs `ariaDescribedBy`.
Not sure if this should be `"invoice-not-found-description"`.

Reply:
Oops, may have mis-copied/pasta'd. Thanks for catching :)